### DISCUSSION TITLE
Flagged account - how to bring it back to normal?

### DISCUSSION THREAD

**Discussion by malykhin-yuri** [community/community#113659] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear Github users and developers!
There is an account on the Github and I personally know its owner. However, the account is "flagged" as spam due to "suspicious activity". How can I help to unset this flag? Where should I write and what proofs that this account is normal, can I provide?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @malykhin-yuri ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
